pkg/controller/rhmiconfig/rhmiconfig_controller.go

```diff
@@ -100,11 +112,71 @@ func (r *ReconcileRHMIConfig) Reconcile(request reconcile.Request) (reconcile.Re
+		logrus.Errorf("failed to reconcile rhmi config values : %v", err)
+		return retryRequeue, err
+	}
+
+	// create cloud resource operator override config map
+	if err := r.ReconcileCloudResourceStrategies(rhmiConfig); err != nil {
+		logrus.Errorf("rhmi config failure while reconciling cloud resource strategies : %v", err)
+		return retryRequeue, err
+	}
+
+	logrus.Infof("rhmi config reconciled successfully")
+	return reconcile.Result{Requeue: true, RequeueAfter: 5 * time.Minute}, nil
+}
+
+// reconciles cloud resource strategies, setting backup and maintenance values for postgres and redis instances
+func (r *ReconcileRHMIConfig) ReconcileCloudResourceStrategies(config *integreatlyv1alpha1.RHMIConfig) error {
+	logrus.Info("reconciling cloud resource maintenance strategies")
+
+	// validate the applyOn and applyFrom values are correct
+	// we also perform this validation on the validate web-hook
+	// we should also provide validation in the controller prior to provisioning/updating the cloud resources (Postgres and Redis)
+	// this is to avoid any chance of a badly formatted value making it to CRO
+	// cloud providers are consistently good at obscure error messages
+	backupApplyOn, maintenanceApplyFrom, err := integreatlyv1alpha1.ValidateBackupAndMaintenance(config.Spec.Backup.ApplyOn, config.Spec.Maintenance.ApplyFrom)
 	if err != nil {
-		return reconcile.Result{}, err
+		return fmt.Errorf("failure validating backup and maintenance values : %v", err)
 	}
 
-	return reconcile.Result{}, nil
+	// build time config expected by CRO
+	timeConfig := &croUtil.StrategyTimeConfig{
+		BackupStartTime:      backupApplyOn,
+		MaintenanceStartTime: maintenanceApplyFrom,
+	}
+
+	// reconcile cro strategy config map, RHMI operator does not care what infrastructure the cluster is running in
+	// as we support different cloud providers this CRO Reconcile Function will ensure the correct infrastructure strategies are provisioned
+	if err := croUtil.ReconcileStrategyMaps(r.context, r.client, timeConfig, croUtil.TierProduction, config.Namespace); err != nil {
+		return fmt.Errorf("failure to reconcile aws strategy map : %v", err)
+	}
+
+	return nil
+}
+
+// we require that blank applyOn and applyFrom values be set to defaults
+// we expect a user to set their own times, but in the case where times are not set
+// we set our maintenance applyFrom values to be Thu 02:00
+// we set out backup applyOn values to be 03:01
+func (r *ReconcileRHMIConfig) reconcileBackupAndMaintenanceValues(rhmiConfig *integreatlyv1alpha1.RHMIConfig) error {
+	if _, err := controllerutil.CreateOrUpdate(r.context, r.client, rhmiConfig, func() error {
+		if rhmiConfig.Spec.Maintenance.ApplyFrom == "" {
+			rhmiConfig.Spec.Maintenance.ApplyFrom = integreatlyv1alpha1.DefaultMaintenanceApplyFrom
+		}
+		if rhmiConfig.Spec.Backup.ApplyOn == "" {
+			rhmiConfig.Spec.Backup.ApplyOn = integreatlyv1alpha1.DefaultBackupApplyOn
+		}
```

Comment:
This defaulting logic could be added as well to the mutating webhook, which is already implemented to add metadata, but can also be used to mutate the spec:
https://github.com/ciaranRoche/integreatly-operator/blob/intly-7203/pkg/apis/integreatly/v1alpha1/rhmiconfig_types.go#L168-L189